resources/views/dashboard.blade.php

```diff
@@ -114,8 +113,9 @@
             connect() {
                 this.pusher = new Pusher(this.app.key, {
                     wsHost: this.app.host === null ? window.location.hostname : this.app.host,
-                    wsPort: this.port,
-                    wssPort: this.port,
+                    wsPort: this.app.port === null ? 6001 : this.app.port,
+                    wssPort: this.app.port === null ? 6001 : this.app.port,
```

Comment:
Looks like duplicate line and with wrong variable name `wssPort` instead of `wsPort`

Reply:
No, both `wsPort` and `wssPort` are needed, `wsPort` is for unsecure connection and `wssPort` is for secure connection